app/assets/stylesheets/v2/_forms.scss

```diff
@@ -0,0 +1,27 @@
+.form-control:disabled {
```

Comment:
> DEPRECATION WARNING on line 127 of app/assets/stylesheets/v1/moj/_forms.scss:
> Extending a compound selector, .form-control:disabled, is deprecated and will not be supported in a future release.
> See https://github.com/sass/sass/issues/1599 for details.

Should we take any action on this? Ideally we should not add new styles that are already considered deprecated.